sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -801,7 +803,7 @@ def _add_argparse_args(cls, parser):
               '"<ENV_VAL>"} }. All fields in the json are optional except '
               'command.'))
     parser.add_argument(
-        '--sdk_worker_parallelism', default=None,
+        '--sdk_worker_parallelism', default=1,
```

Comment:
Do we have strong reasons not to make 0 be a default value? Folks working on portability probably have an informed opinion on this. cc: @mxm, @robertwb.

Reply:
From PortablePipelineOptions.java: 

```  @Description(
      "Sets the number of sdk worker processes that will run on each worker node. Default is 1. If"
          + " 0, it will be automatically set by the runner by looking at different parameters "
          + "(e.g. number of CPU cores on the worker machine).")```

Reply:
I agree, we probably want `0`.

Reply:
I have seen machine lock down when running python processes equal to the number of cores. 
We can still go with 0 here as long as we set the default to be 1 in JobServerDriver.java.
And I think this is reasonable.